Crane/app/src/main/java/androidx/compose/samples/crane/util/UnsplashSizingInterceptor.kt

```diff
@@ -50,11 +38,9 @@ object UnsplashSizingInterceptor : Interceptor {
         val size = chain.size
         if (data is String &&
             data.startsWith("https://images.unsplash.com/photo-") &&
-            size is PixelSize &&
-            size.width > 0 &&
```

Comment:
`PixelSize` is guaranteed to have a positive width/height.

Reply:
ack—thanks!